### ISSUE TITLE
how to get list text?

### ISSUE THREAD

**Issue by YeonwooJeong** [microsoft/WinAppDriver#1733]
I tried to get all the file names on this list.

![image](https://user-images.githubusercontent.com/8816386/173298037-d5e50ede-cea5-46c6-bde5-062d0bc68034.png)

xpath
String folder_listview = "/Pane[@ClassName=\"#32769\"][@Name=\"데스크톱 1\"]"
    			+ "/Window[@ClassName=\"CabinetWClass\"][@Name=\"snyc2\"]"
    			+ "/Pane[@ClassName=\"ShellTabWindowClass\"][@Name=\"sync2\"]"
    			+ "/Pane[@ClassName=\"DUIViewWndClassName\"]"
    			+ "/Pane[@Name=\"셸 폴더 보기\"][@AutomationId=\"listview\"]"
    			+ "/List[@ClassName=\"UIItemsView\"][@Name=\"항목 보기\"]"
    			+ "/ListItem[@ClassName=\"UIItem\"]";

Why can't read the list? It was a code that worked normally on other routes.

my code
`   	
String folder_listview = "/Pane[@ClassName=\"#32769\"][@Name=\"데스크톱 1\"]"
    			+ "/Window[@ClassName=\"CabinetWClass\"][@Name=\"snyc2\"]"
    			+ "/Pane[@ClassName=\"ShellTabWindowClass\"][@Name=\"sync2\"]"
    			+ "/Pane[@ClassName=\"DUIViewWndClassName\"]"
    			+ "/Pane[@Name=\"셸 폴더 보기\"][@AutomationId=\"listview\"]"
    			+ "/List[@ClassName=\"UIItemsView\"][@Name=\"항목 보기\"]"
    			+ "/ListItem[@ClassName=\"UIItem\"]";
    	
    	
    	System.out.println("---------1---------\n");
    	List<WebElement> sync_list_elements = (List<WebElement>) winapp.findElementsByXPath(folder_listview);
		int size = sync_list_elements.size();
		System.out.println(size);
		//목록 모두 list에 추가하기
		for (WebElement e : sync_list_elements) {
			list.add(e.getText());
			System.out.println(e.getText());
		}
		System.out.println("------------------\n");`

https://user-images.githubusercontent.com/8816386/173298871-4a8669bc-d9fb-4a6f-bf26-96c1d1f06ed7.mp4

**Comment by YeonwooJeong** [comment]
> Why are you using winappdriver for this? You can get the list programmatically?

yes,I'm going to compare the list of local files with the list of files on the server.

**Comment by liljohnak** [comment]
https://docs.microsoft.com/en-us/dotnet/api/system.io.directory.getfiles?view=net-6.0
